segments/weather.sh

```diff
@@ -1,7 +1,7 @@
 # Prints the current weather in Celsius, Fahrenheits or lord Kelvins. The forecast is cached and updated with a period of $update_period.
 
 # The update period in seconds.
-update_period=600
+# update_period=600
```

Comment:
Hm it looks like this one is not used anymore, right? If so we could remove these 2 lines.

Reply:
roger that~